README.md

```diff
@@ -1 +1,133 @@
+    <br />
+    <a href="https://github.com/blobbyblobfish/piggy-bank/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/blobbyblobfish/piggy-bank/issues">Request Feature</a> 
+  </p>
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Built With](#built-with)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+* [Usage](#usage)
+* [Roadmap](#roadmap)
+* [Contributing](#contributing)
+* [License](#license)
+* [Contact](#contact)
+* [Acknowledgements](#acknowledgements)
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+[![Product Name Screen Shot][product-screenshot]](https://www.example.com/)
+
+This application was built with a rails backend and Javascript frontend. 
+
+### Built With
+* [Ionic](https://ionicframework.com/)
+* [React](https://reactjs.org/)
+* [MongoDB](https://www.mongodb.com/)
+* [Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To get the application started you must have npm and mongoDB installed on your computer. Checkout the npm [documentation](https://www.npmjs.com/) to learn how to install it on your device. If you need to install mongoDB check out the doumentation [here](https://www.mongodb.com/).
+
+### Prerequisites
+You will need to clone the backend for this application using the following repo: <a href="https://github.com/blobbyblobfish/piggy-bank-backend"> piggy-bank-backend </a> 
+
+### Installation
+
+1. Clone the repo
+```sh
+git clone git@github.com:blobbyblobfish/piggy-bank.git
+```
+2. 
+```sh
+npm install 
+```
+3. 
+```sh
+npm start
+```
+
+<!-- ROADMAP -->
+## Roadmap
+
+See the [open issues](https://github.com/blobbyblobfish/piggy-bank/issues) for a list of proposed features (and known issues).
+
+<!-- CONTRIBUTING -->
+## Contributing
+
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
+<!-- LICENSE -->
+## License
+
+Distributed under the MIT License. See `LICENSE` for more information.
+
+## Contact
+Connect/Follow us:
+
+Brittany:
+[![Twitter][twitter-shield]][brittany-twitter-url]
+
+Alice:
```

Comment:
```suggestion
[![LinkedIn][linkedin-shield](https://linkedin.com/in/aliceyunwang)]
```

Reply:
@SincerelyBrittany I added my LinkedIn. Could you let me know if this syntax works?